pkg/server/http/option.go

```diff
@@ -6,17 +6,19 @@ import (
 	"github.com/owncloud/ocis-graph/pkg/config"
 	"github.com/owncloud/ocis-graph/pkg/metrics"
 	"github.com/owncloud/ocis-pkg/log"
+	"github.com/owncloud/ocis-pkg/middleware"
 )
 
 // Option defines a single option function.
 type Option func(o *Options)
 
 // Options defines the available options for this package.
 type Options struct {
-	Logger  log.Logger
-	Context context.Context
-	Config  *config.Config
-	Metrics *metrics.Metrics
+	Logger        log.Logger
+	Context       context.Context
+	Config        *config.Config
+	Metrics       *metrics.Metrics
+	OpenIDConnect middleware.OIDCOptions
```

Comment:
Adding this to the options doesn't really make sense? :)

Reply:
Why? These are the config options of openid connect - these need to be set. 
If not here - where else could that be?

Reply:
but you won't add middleware options to the http server package. the middleware gets used within this package, but the command package or other packages using the http server should not even know about any middleware package.

Reply:
And where do I take the config from then? Just let me know and I implement this accordingly. THX

Reply:
@tboerger So where do we configure the middleware? ocis-hello does it in https://github.com/owncloud/ocis-hello/blob/master/pkg/server/http/server.go#L46-L61

Reply:
@butonic what about https://github.com/owncloud/ocis-graph/blob/master/pkg/server/http/server.go#L30-L43?